backend/frontend/src/pages/story/CreateStory.js

```diff
@@ -32,14 +33,25 @@ function CreateStory() {
   const autocompleteRef = useRef(null);
   const inputRef = useRef(null);
 
+  Quill.register('modules/imageCompress', ImageCompress);
+
   const modules = {
-    toolbar: [
-      [{ header: [1, 2, false] }],
-      ["bold", "italic", "underline", "strike", "blockquote"],
-      [{ list: "ordered" }, { list: "bullet" }],
-      ["link", "image", "video"]
-    ]
-  };
+    toolbar: {
+      container: [
+        [{ header: [1, 2, false] }],
+        ["bold", "italic", "underline", "strike", "blockquote"],
+        [{ list: "ordered" }, { list: "bullet" }],
+        ["link", "image", "video"],
+      ],
+    },
+    imageCompress: {
+      quality: 0.7, // default
+      maxWidth: 1024, // default
+      maxHeight: 1024, // default
+      imageType: 'image/jpeg', // default
+      debug: false, // default
+    }
+};
```

Comment:
@ayhncvdr hi,
Here are the image compression lines done by @ckertam

Reply:
I am not sure if we should set max width and height, does this affect the aspect ratio of the images?

Reply:
Well, actually, in @ckertam 's example he got 768x1024 resolution. This module keeps the ratio.

Reply:
It's resolved I guess. We decided to use this with @ayhncvdr2 and we will also do it in mobile part! 🚀